### PR TITLE
Bootstrap snippet gen for PHP.

### PR DESCRIPTION
**Pull request by shinfan**
- Bootstrap PHP Language provider.
- Bootstrap PHP Discovery context.
- Currently the generated snippets are empty.

### CODE REVIEW

File: vgen/src/main/resources/io/gapi/vgen/php/php_discovery_gapic.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+type: io.gapi.vgen.ConfigProto
```
Comment (garrettjonesgoogle) [inline vgen/src/main/resources/io/gapi/vgen/php/php_discovery_gapic.yaml]: Remove "_gapic" from this filename

File: vgen/src/main/java/io/gapi/vgen/php/PhpDiscoveryLanguageProvider.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      throws IOException {
+    Api api = getService().getApis(0);
+    String root =
+        "autogenerated/"
+            + api.getName()
+            + "/"
+            + api.getVersion()
+            + "/"
+            + context.getApiRevision()
+            + "/";
```
Comment (garrettjonesgoogle) [inline vgen/src/main/java/io/gapi/vgen/php/PhpDiscoveryLanguageProvider.java]: Thomas is moving the line to compose this string into DiscoveryContext (see https://github.com/googleapis/toolkit/pull/48/files)

Comment (shinfan) [inline vgen/src/main/java/io/gapi/vgen/php/PhpDiscoveryLanguageProvider.java]: Acknowledged. I will switch to use the one in DiscoveryContext after Thomas's PR is merged.

File: vgen/src/main/resources/io/gapi/vgen/php/php_discovery_gapic.yaml (hunk 1 above)
Comment (shinfan) [inline vgen/src/main/resources/io/gapi/vgen/php/php_discovery_gapic.yaml]: Right now this naming pattern is consistent with Ruby and Java. If possible, I would prefer to leave it as-is for now and address this for all languages in a separate PR.

Comment (garrettjonesgoogle) [inline vgen/src/main/resources/io/gapi/vgen/php/php_discovery_gapic.yaml]: This rename is part of Thomas' change, so if you submit before he does, then he would want to add the rename into his own PR; if you submit after, then you would want to include the rename.

**Comment by garrettjonesgoogle** [comment]
LGTM

**Pull request merged**